DESCRIPTION

```diff
@@ -30,27 +30,29 @@ Imports:
     assertthat,
     curl,
     fields,
+    frictionless,
     ggplot2,
     glue,
     graphics,
     lubridate,
     lutz,
-    maptools,
+    suntools,
     methods,
     raster,
     readr,
     dplyr (>= 1.1.0),
     rhdf5,
     sp,
-    sf,
     stats,
     tidyr,
     utils,
     viridisLite,
     viridis,
-    rlang
+    rlang,
+    jsonlite
 Remotes:
-    bioc::rhdf5
+    bioc::rhdf5,
+    github::adokter/suntools
```

Comment:
github remotes can be removed

Reply:
fixed in [1c9492d](https://github.com/adokter/bioRad/pull/590/commits/1c9492da8569a1156c2ef4ae1c3a71500b5a8c4c)